civis/io/_files.py

```diff
@@ -268,11 +272,12 @@ def _file_to_civis(buf, name, api_key=None, client=None, **kwargs):
     elif file_size > MAX_FILE_SIZE:
         msg = "File is greater than the maximum allowable file size (5TB)"
         raise ValueError(msg)
-    elif not is_seekable and file_size > MAX_PART_SIZE:
-        msg = "Cannot perform multipart upload on non-seekable files. " \
+    elif not multipart_upload and file_size > MAX_PART_SIZE:
+        msg = "Cannot perform multipart upload on non-seekable files or " \
```

Comment:
We do perform multi-part uploads on non-seekable files, don't we? We write those to disk first.

Reply:
True, we now write these to disk. I think this is related to the ^^ comment and discussion.